packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1052,7 +1052,8 @@ enroll.mdm.copyLink = Copy link to clipboard
 enroll.mdm.copyLink.success = Link copied.
 enroll.mdm.step1 = Tap the Copy Link button below.
 enroll.mdm.step2 = On this device, open your browser, then paste the copied link into the address bar.
-enroll.mdm.step3 = Follow the instructions in your browser to set up <span class="semi-strong">{0}</span>, then try accessing this app again.
+enroll.mdm.step3 = Follow the instructions in your browser to set up <span class="semi-strong">{0}</span>.
```

Comment:
Please use our [HTML tag syntax](https://github.com/okta/okta-ui/#handlebars-i18n) to avoid having to embed HTML within the i18n template. As a result, we can now remove the forced un-escape `{{{value}}}` as a part of the template.